zebra/zebra_vty.c

```diff
@@ -738,6 +755,7 @@ DEFPY(ip_route_blackhole_vrf,
 	  tag (1-4294967295)                                                  \
 	  |(1-255)$distance                                                   \
 	  |label WORD                                                         \
+	  |table (1-4294967295)                                               \
```

Comment:
This makes absolutely no sense.  A route inside of a vrf that has a different table?  Please remove

Reply:
Is there a specific use case for having multiple tables within a vrf? This seems to add a good bit of complexity from a user's perspective. I'd be hesitant to include this functionality unless we had a specific use case... ??

Reply:
having multple table per vrf is only available with a vrf backend = VRF netns